q2_assembly/bowtie2/indexing.py

```diff
@@ -21,6 +22,19 @@
 from q2_assembly.bowtie2.utils import _get_subdir_from_path, _process_bowtie2build_arg
 
 
+def assert_inputs_not_empty(fasta_fps: list):
```

Comment:
This function looks to me like it could be moved to `bowtie2/utils.py` - it could potentially be used in other submodules (maybe even the `_utils.py` one level above?).